### DISCUSSION TITLE
debian bookworm/wayland regolith3 beta : keybinding viewer, hiding popups, etc.

### DISCUSSION THREAD

**Discussion by clarkevans** [regolith-linux/regolith-desktop#868] [General]
Thank you so much for getting Regolith 3 working on Bookworm/Wayland via Sway. It worked pretty much right out of the box without any serious problems. I'm using today's `https://regolith-desktop.org/unstable-debian-testing-amd64 testing InRelease`
> `Linux napier 6.1.0-9-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.27-1 (2023-05-08) x86_64 GNU/Linux`

I figured that I can put my "wm.binding." overrides in `.config/regolith3/Xresources`, and I found the [reference](https://regolith-desktop.com/docs/reference/xresources/) for this.  Thank you.  It's pretty easy to configure.

Here are some things I notice with the beta.

1. The keybinding viewer, e.g. Shift+? doesn't seem to work.
2. I have a red disk icon at 16% on my status bar, and I'm unsure what that is (or how to interact with status bar).
3. MS Teams application (used with Edge) and Zoom both use nasty pop-ups for notifications... how do I make them float?
4. I'm still not sure about the relation of Regolith to Sway and Wayland, e.g. how to configure things more precisely
5. The logout key bindings don't seem to be working; is there a way to logout and/or reboot (besides `sudo reboot`)

Thank you so much for Regolith.

**Comment by cfsmp3** [comment]
1 - Most likely you don't have a US layout. You need to configure the binding so it uses the primary character (the one that is on the ? key that appears when you press it without shift). 

For example, if you keyboard was Spanish, in 
/usr/share/regolith/common/config.d/20_ilia

You'd have this:
```
## Launch // Keybinding Viewer // <><Shift> ? ##
set_from_resource $wm.binding.help wm.binding.help Shift+apostrophe
set_from_resource $wm.program.help wm.program.help ilia -p keybindings
bindsym $mod+$wm.binding.help exec --no-startup-id $wm.program.help -a -t $ilia.stylesheet
```
(note apostrophe instead of ?)
This is a sway thing, not a regolith thing.

2 - What status bar is it? (don't know if it's already i3status-rs). 

3 - This might get you in the right direction
https://www.reddit.com/r/swaywm/comments/l9asbc/zoom_on_sway_what_settings_do_you_use/

I haven't tried personally, but if it does work let us know. If not, do come to slack. Happy to work with you into finding a working "official" recommended settings. Maybe we could just have a regolith-sway-zoom-config package for everyone's convenience.

4 - Still getting to know each other :-) I've been using it for maybe 9 months (daily driver) and it works like 98% perfectly. The major issues you've already discovered. 

5 - Most likely same as 1. 

I'd say, for the amount of people working on this (pretty much @SoumyaRanjanPatnaik doing all the work on sway, @kgilmer doing 95% of the packages, and myself just nagging them and behaving like a regular user instead of a developer :-) ) things are looking quite good.  I'm confident we'll get more users reporting problems but also some valuable pull requests and new contributors to our sway journey.
